overrides/cf.d.ts

```diff
@@ -183,12 +183,26 @@ interface RequestInitCfPropertiesImageDraw extends BasicImageTransformations {
   right?: number;
 }
 
+interface RequestInitCfPropertiesImageTrim extends BasicImageTransformations {
+  left?: number;
+  top?: number;
+  right?: number;
+  bottom?: number;
+}
+
 interface RequestInitCfPropertiesImage extends BasicImageTransformations {
   /**
    * Device Pixel Ratio. Default 1. Multiplier for width/height that makes it
    * easier to specify higher-DPI sizes in <img srcset>.
    */
   dpr?: number;
+  /**
+   * An object with four properties {left, top, right, bottom} that specify
+   * a number of pixels to cut off on each side. Allows removal of borders
+   * or cutting out a specific fragment of an image. Trimming is performed
+   * before resizing or rotation. Takes dpr into account.
+   */
+  trim?: RequestInitCfPropertiesImageTrim;
```

Comment:
Can you inline this as well please? The trim properties are not re-used anywhere.

Reply:
I followed `RequestInitCfPropertiesImageDraw`, that isn't used either... but if you prefer I can do the change.

Reply:
Yeah, `RequestInitCfPropertiesImageDraw` isn't reused, but it's its own interface so it can extend `BasicImageTransformations`. `trim` doesn't need that :)

Reply:
Agreed then, will fix.